drivers/pwm/pwm_b91.c

```diff
@@ -6,18 +6,19 @@
 
 #define DT_DRV_COMPAT telink_b91_pwm
 
-#include "pwm.h"
 #include "clock.h"
-#include <zephyr/drivers/pwm.h>
-#include <zephyr/drivers/pinctrl.h>
+#include "pwm.h"
 
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/drivers/pwm.h>
 
-#define PWM_PCLK_SPEED      DT_INST_PROP(0, clock_frequency)
-#define NUM_OF_CHANNELS     DT_INST_PROP(0, channels)
+#define PWM_PCLK_SPEED	DT_INST_PROP(0, clock_frequency)
+#define NUM_OF_CHANNELS DT_INST_PROP(0, channels)
 
 PINCTRL_DT_INST_DEFINE(0);
 
-static const struct pinctrl_dev_config *pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(0);
+static const struct pinctrl_dev_config *pcfg =
+	PINCTRL_DT_INST_DEV_CONFIG_GET(0);
```

Comment:
Picking one change at random here: What is the gain of reformatting all the files unconditionally? The code does not get more readable by this.

Tools like `clang-format` can only work on a set of rules, but does not perceive code as the human brain does. Alignment of definitions on consecutive lines, if clauses, ... makes sense to the human brain and helps it see relationships, and aids in humans being able to scan the code quickly.

Reply:
FWIW, the Linux kernel process guidelines do not recommend blindly using `clang-format` to reformat either: https://www.kernel.org/doc/html/latest/process/clang-format.html

Reply:
Auto-formatters are not perfect, this is why they can be turned off on certain regions if needed. Other languages have done a much better job at this from the beginning (e.g. Rust) and others like Python have great tools nowadays (e.g. black), C as usual is years behind. By looking at the Zephyr codebase, I still think an auto-formatter will help on areas like formatting device definition macros (it's now a mess). Auto-formatters are also useful when refactoring (it is painful without). The line above probably indicates that 80 cols is not the smartest choice for Zephyr. Maybe increasing to 100, or 88 (like did `black` in Python after a study) will yield better results.

Reply:
> FWIW, the Linux kernel process guidelines do not recommend blindly using `clang-format` to reformat either: [kernel.org/doc/html/latest/process/clang-format.html](https://www.kernel.org/doc/html/latest/process/clang-format.html)

Linux Kernel coding style is loosely specified, that's probably why. `checkpatch` just checks a few things, but one can write code in subtle different styles while still making it pass... Linux is great in many aspects, but when it comes to coding consistency or documentation... there are better examples around, really.